chapters/infrastructure/collaboration/reading/read.md

```diff
@@ -1,12 +1,93 @@
+
+There are three [roles](../../../building-blocks/roles/reading/read.md) involved in collaborative content contribution activities: content developers, contributors and content curators.
+Contributions follow the [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow).
+
+Contributors make submissions with proposed changes to content.
+These changes may be fixes, improvements or completely new content.
+Contributors create their own work as Git commits in GitHub branches / forks and then submit it to the repository.
+The are submitted as [GitHub pull requests](https://docs.github.com/en/pull-requests) to be reviewed.
+
+Contributors can also submit [GitHub issues](https://docs.github.com/en/issues) that can trigger content contributions.
+These are to be discussed and, at some point, assigned to a contributor.
+
+Content developers are usually the original creators of the repository.
+They provide the initial content repository and, ideally, they will be actively part of the community.
+Content developers most often fill the role of reviewer of submissions as part of GitHub pull request discussions;
+they vet the submission, provide feedback and assist in getting the submission to a high quality state.
+
+Content curator oversee the general quality of the repository and ensure there is a stable community around it, with constant contributors and a healthy review process from content developers.
+Content curators may take an active role in reviewing (and contributing).
+But, primarily, they would ensure that submissions are reviewed in a timely manner.
+Moreover, they are concerned with the building of a community with good interaction between members, as presented in [the "Curating and Maintain Content" chapter](../../../curate-maintain/overview/reading/read.md).
+
+Contributions happen both in the content repository (hosted by Open Education Hub) and the delivery repository (the forked version used by educators).
+The workflow is similar.
+For the delivery repository, educators act as content developer / reviewers.
+It is the responsibility of the content curator to ensure that content in delivery repositories makes its way to corresponding content repositories.
+
+In order to assist the contribution workflow, guidelines are in place for each repository.
+The template repositories ([`oer-template`](https://github.com/open-education-hub/oer-template) and [`oer-internal-template`](https://github.com/open-education-hub/oer-internal-template/)) feature [contributing guidelines](https://github.com/open-education-hub/oer-template/blob/main/CONTRIBUTING.md) and [reviewing guidelines](https://github.com/open-education-hub/oer-template/blob/main/REVIEWING.md).
+Also templates for pull requests and issues are part of the template [GitHub Actions workflows](https://github.com/open-education-hub/oer-template/tree/main/.github/workflows).
```

Comment:
```suggestion
Also templates for pull requests and issues are part of the template [GitHub Actions workflows](https://github.com/open-education-hub/oer-template/tree/main/.github).
```
And in fact, we don't have an issue template, only a PR template [1]. We can create one though.

[1] https://github.com/open-education-hub/oer-template/blob/main/.github/pull_request_template.md

Reply:
I just added there issue for completion. We can create one later on.